frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -256,24 +256,28 @@ export type CreateDashboardRequest = {
 };
 
 export type UpdateDashboardRequest = {
-  id: DashboardId;
-  parameters?: Parameter[] | null;
-  point_of_interest?: string | null;
-  description?: string | null;
-  archived?: boolean | null;
-  dashcards?: DashboardCard[] | null;
   collection_position?: number | null;
-  tabs?: DashboardTab[];
-  show_in_getting_started?: boolean | null;
-  enable_embedding?: boolean | null;
-  collection_id?: CollectionId | null;
-  name?: string | null;
-  width?: DashboardWidth | null;
   caveats?: string | null;
-  embedding_params?: EmbeddingParameters | null;
-  cache_ttl?: number;
   position?: number | null;
-};
+} & Pick<Dashboard, "id"> &
```

Comment:
nit: I think we can keep `DashboardId` in place of `Pick<Dashboard, "id">` if they are the same.